lib/cmd/fh3/app/logs/tail.js

```diff
@@ -55,11 +101,12 @@ module.exports = {
         }
         if (msg.data && msg.data.length !== 0) {
           var logString = msg.data.join("\n");
-          console.log(logString);
+          console.log(params.logname + ":");
+          console.log(logString+"\n");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this logging statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3AS2228)